src/compute/src/logging/compute.rs

```diff
@@ -709,51 +779,62 @@ impl<A: Allocate> DemuxHandler<'_, '_, A> {
 
     /// Handle the given compute event.
     fn handle(&mut self, event: ComputeEvent) {
-        use ComputeEvent::*;
-
         match event {
-            Export { id, dataflow_index } => self.handle_export(id, dataflow_index),
-            ExportDropped { id } => self.handle_export_dropped(id),
-            Peek {
+            ComputeEvent::Export(Export { id, dataflow_index }) => {
+                self.handle_export(id, dataflow_index)
```

Comment:
Nit: I think here we should just give the event structs to the handler methods and then do the unpacking in the handler.

```rust
match event {
    ComputeEvent::Export(event) => self.handle_export(event),
    ComputeEvent::ExportDropped(event) => self.handle_export_dropped(event),
    ComputeEvent::Peek(event) => self.handle_peek(event),
    ...
}
```

Makes the match a bit less noisy, especially for events with many fields.

Reply:
I added a commit that does what you're suggesting, and in the course of implementing it I realized we were not consistent in the names we'd use to describe the same thing. So, there's a bunch of renaming part of this PR, which should really be no functional change. I hope.